services/wallet/api.go

```diff
@@ -742,11 +753,91 @@ func (api *API) CreateMultiTransaction(ctx context.Context, multiTransactionComm
 	return nil, api.s.transactionManager.SendTransactionForSigningToKeycard(ctx, cmd, data, api.router.GetPathProcessors())
 }
 
+func (api *API) BuildTransactionsFromRoute(ctx context.Context, uuid string, slippagePercentage float32) (err error) {
```

Comment:
we can make this API a bit more break-proof by using a struct (`parameters` or something) instead of plainly `slippagePercentage`

Reply:
@dlipicar agree, good idea, done.